manifests/profile/server.pp

```diff
@@ -288,6 +292,34 @@
     tag     => 'st2::config',
   }
 
+  ## Metrics Settings
```

Comment:
Instead of making this part of `profile/server.pp`, can we extract it out to another component, say `server/monitoring.pp`?